CMakeLists.txt

```diff
@@ -373,6 +373,8 @@ include(nuttx_add_application)
 include(nuttx_add_romfs)
 include(nuttx_add_symtab)
 include(nuttx_add_module)
+include(nuttx_add_dependencies)
+include(nuttx_apps_headers)
```

Comment:
could we make more general name since apps is replaceable.

Reply:
Yes, do you have any suggestion💡
how about `nuttx_include_directories` ? that is nuttx wrapper name for cmake module `include_directories`.
or `nuttx_export_header` ? that expresses its function.

Reply:
> could we make more general name since apps is replaceable.

Done, changed module name to `nuttx_export_header.cmake`